### PR TITLE
remove percentages from compile time constants

### LINKED ISSUE #5: Fee contract
- The fee distribution contract must be implemented 
- Must distribute to the original project creators and the lilium devs

### PR DESCRIPTION
**Pull request by mgpai22**

### CODE REVIEW

File: contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es [hunk 1]
```diff
@@ -20,19 +20,20 @@
...
-    // $phoenixPercentage: (Long, Long)
     // $minerFee: Long
-  
+
     // ===== Context Variables (@) ===== //
     // None
 
+    val devPercentage: (Long, Long) = (60L, 100L)
+    val phoenixPercentage: (Long, Long) = (40L, 100L)
+
```
Comment (ldgaetano) [inline contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es:31]: just make feeDenom one value as 100 and change devPercentage to devPercentageNum and phoenixPercentage as phoenixPercentageNum

Comment (ldgaetano) [reply contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es:31]: we should talk to them to agree on a value so we only need to do this once

File: contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es [hunk 2]
```diff
@@ -20,19 +20,20 @@
...
     // ===== Relevant Variables ===== //
     val dev1Address: SigmaProp                  = PK("9hHondX3uZMY2wQsXuCGjbgZUqunQyZCNNuwGu6rL7AJC8dhRGa")
     val dev2Address: SigmaProp                  = PK("9gnBtmSRBMaNTkLQUABoAqmU2wzn27hgqVvezAC9SU1VqFKZCp8")
     val dev3Address: SigmaProp                  = PK("9iE2MadGSrn1ivHmRZJWRxzHffuAk6bPmEv6uJmPHuadBY8td5u")
     val phoenixAddress: SigmaProp               = PK("9iPs1ujGj2eKXVg82aGyAtUtQZQWxFaki48KFixoaNmUAoTY6wV")
-    val minerFeeErgoTreeBytesHash: Coll[Byte]   = fromBase16("2b9e147dda83b66925c7718dd40f7df43482e1689ce53363923b2fe3908952a9")
+    val minerFeeErgoTreeBytesHash: Coll[Byte]   = fromBase16("e540cceffd3b8dd0f401193576cc413467039695969427df94454193dddfb375")
```
Comment (ldgaetano) [inline contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es:37]: is this the actual ergotree hash ???

Comment (mgpai22) [reply contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es:37]: e540... is the miner ergotree hash

File: contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es (hunk 1 above)
Comment (mgpai22) [reply contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es:31]: yes

Comment (ldgaetano) [reply contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es:31]: also maybe make feedenom as 1000 and numerator as 600 and 1000, good to test this actually to make sure values are large enough

**Pull request merged**